### DISCUSSION TITLE
New project

### DISCUSSION THREAD

**Discussion by Mongskl** [community/community#118687] [Programming Help]
Ive recently started html,css,js. I wanna try to make some simple projects but I have no idea where to start. Can anyone give me some advice like what kind of stuff I should work on?

**Comment by DeathKnight813** [comment]
First master html,css,js in w3 and next try to make simple projects.
Try is the best way to learning.

**Comment by rsr077** [comment]
> Ive recently started html,css,js. I wanna try to make some simple projects but I have no idea where to start. Can anyone give me some advice like what kind of stuff I should work on?

You just start GitHub .

**Comment by Bl4ck-H4ck3r** [comment]
1.Personal Portfolio Page
2.eCommerce Page
3.Small Business Homepage
4.Event Invitation Page
and ...
